### PR TITLE
[TS] LPS-124521

### PR DESCRIPTION
**Pull request by jonathanmccann**
/cc @brianikim

### CODE REVIEW

**Comment by jonathanmccann** [comment]
ci:test:sf

**Comment by jonathanmccann** [comment]
ci:test:relevant

File: modules/apps/frontend-css/frontend-css-web/src/main/resources/META-INF/resources/taglib/_search_iterator.scss [hunk 1]
```diff
@@ -138,6 +138,10 @@
 	.lfr-asset-item {
 		width: 50%;
 
+		@include media-breakpoint-up(xs) {
```
Comment (pat270) [inline modules/apps/frontend-css/frontend-css-web/src/main/resources/META-INF/resources/taglib/_search_iterator.scss:141]: We can get rid of this whole block and just change `width: 50%;` => `width: 100%;` The media query generated by `@include media-breakpoint-up(xs)` is `@media (min-width: 0)`.

**Review by pat270 — CHANGES_REQUESTED** [review]

**Comment by jbalsas** [comment]
Hey @jonathanmccann, @brianikim, thanks for sending this over!

In order to improve the quality and consistency of our review process we'd appreciate if you could read through the following reference guides on what we expect of PRs and the PR review process:
- [Frontend Infrastructure Team code review process](https://github.com/liferay/liferay-frontend-guidelines/blob/master/dxp/code_review.md)
- [Pull Request Etiquette](https://github.com/liferay/liferay-frontend-guidelines/blob/master/general/pull_requests.md)

In summary, we find that the following greatly helps us with this process:
- Provide a clear title for a PR. Avoid simply stating a JIRA ticket in it and be as descriptive as possible
- Provide a clear description of what the PR is doing and the problem it's solving (even if it's already in the JIRA ticket). This helps us get a clear context quickly and keep track of it when the PR is forwarded.
- Provide a test plan so we can easily know how to test the changes or be comfortable that it's already been done thoroughly
- Add any additional information you think important (things you tried, things that didn't work, analysis of the problem...)

We really appreciate your help in improving this process for everyone!! 🤗

**Comment by brianikim** [comment]
https://github.com/liferay-frontend/liferay-portal/pull/623

**Comment by jbalsas** [comment]
Thanks a lot @brianikim! In the future (at least when dealing with our team) feel free to simply work on the same PR, no need to redo them. We're fine iterating on the same PR until we're all happy with it as a way to simplify keeping track of work.

**Pull request closed without merging**